airbyte-db/lib/src/test/java/io/airbyte/db/check/impl/ConfigsDatabaseAvailabilityCheckTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.check.impl;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.jooq.DSLContext;
+import org.jooq.Select;
+import org.jooq.exception.DataAccessException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test suite for the {@link ConfigsDatabaseAvailabilityCheck} class.
+ */
+public class ConfigsDatabaseAvailabilityCheckTest extends AbstractDatabaseAvailabilityCheckTest {
+
+  @Test
+  void checkDatabaseAvailability() {
+    final var check = new ConfigsDatabaseAvailabilityCheck(dslContext, 60000L);
```

Comment:
Does this mean that it takes 60 seconds for this test to run? If so, that seems very high to me. Is there real value in having this be set so high rather than something low like `200 ms`?

Reply:
This can be lower for testing purposes.